deployer/src/deployer/search/__init__.py

```diff
@@ -347,7 +347,7 @@ def analyze(
 ):
     # We can confidently use a single host here because we're not searching a cluster.
     connections.create_connection(hosts=[url])
-    index = Document._index
+    index = Index("mdn_docs")
```

Comment:
This is pretty unrelated. But when we started using the dynamic indexes the `poetry run deployer search-analyze "forEach"` CLI had stopped working. I needed that to work on this PR so I included this change.